### PR TITLE
Add clearTimedEvent() and clearTimedEvents()

### PR DESCRIPTION
**Pull request by zihejia**
addresses #740

### CODE REVIEW

File: src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java [hunk 1]
```diff
@@ -502,6 +502,29 @@ public void timeEvent(final String eventName) {
...
+        synchronized (mEventTimings) {
+            mEventTimings.clear();
+            mPersistentIdentity.clearTimedEvents();
+        }
+    }
+
+    /**
+     * Clears the event timing for an event.
+     *
+     * @param eventName the name of the event to track with timing.
```
Comment (jaredmixpanel) [inline src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java:519]:
```suggestion
     * @param eventName the name of the timed event to clear.
```

File: gradle.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=6.0.0
+VERSION_NAME=6.0.1-SNAPSHOT
```
Comment (jaredmixpanel) [inline gradle.properties:1]: okay to have `-SNAPSHOT` here in master?

**Review by jaredmixpanel — APPROVED** [review]
lgtm w/ one question and one suggestion

File: gradle.properties (hunk 2 above)
Comment (zihejia) [reply gradle.properties:1]: yes, it should. I will figure out why the release script did not change it.

**Pull request merged**